patroni/postgresql/__init__.py

```diff
@@ -610,7 +631,7 @@ def wait_for_startup(self, timeout=None):
 
         return self.state == 'running'
 
-    def restart(self, timeout=None, task=None, block_callbacks=False, role=None):
+    def restart(self, timeout=None, task=None, block_callbacks=False, role=None, stop_timeout=None):
```

Comment:
The `restart()` is never called with `stop_timeout` argument.

Reply:
Yes, missed out on removing the reference after code update. will fix it.

Reply:
```suggestion
    def restart(self, timeout=None, task=None, block_callbacks=False, role=None):
```